.github/workflows/deploy.yml

```
@@ -187,8 +157,12 @@
           echo "\$webhooksApiUrl   = \"${{ secrets.WEBHOOKS_API_URL }}\";" >> webhooks.secrets.php
           echo "\$webhooksApiToken = \"${{ secrets.WEBHOOKS_API_TOKEN }}\";" >> webhooks.secrets.php
 
-      - name: Zip files
-        if: steps.changes.outputs.src == 'true' || github.event_name == 'workflow_dispatch'
+      - name: 📝 Create Version File for API
+        run: |
+          cd Src       
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)